app/controllers/works_controller.rb

```diff
@@ -244,7 +254,7 @@ def work_params
                      :abstract, :citation_auto, :citation, :default_citation,
                      :access, :license, :version_description,
                      :release, 'embargo_date(1i)', 'embargo_date(2i)', 'embargo_date(3i)',
-                     :agree_to_terms, :assign_doi, :upload_type, :globus, :fetch_globus_files,
+                     :agree_to_terms, :assign_doi, :upload_type, :globus, :fetch_globus_files, :attached_files_list,
```

Comment:
Why is this getting added?

Reply:
Argh, left over from a previous approach. Thanks for catching.